assets/js/data/validation/test/reducers.ts

```diff
@@ -155,6 +155,26 @@ describe( 'Validation reducer', () => {
 		expect( nextState ).toHaveProperty( 'testError' );
 	} );
 
+	it( 'Clears multiple validation errors', () => {
+		const state: Record< string, FieldValidationStatus > = {
+			existingError: {
+				message: 'This is an existing error message',
+				hidden: false,
+			},
+			testError: {
+				message: 'This is error should not be removed',
+				hidden: false,
+			},
+		};
+		const clearError: ValidationAction = {
+			type: types.CLEAR_VALIDATION_ERRORS,
+			errors: [ 'existingError', 'testError' ],
+		};
+		const nextState = reducer( state, clearError );
+		expect( nextState ).not.toHaveProperty( 'existingError' );
+		expect( nextState ).not.toHaveProperty( 'testError' );
```

Comment:
It's a bit confusing that the message of this error is `This is error should not be removed` but here you're checking that it has been removed.

Reply:
Right! Good catch, thanks Alex.